gbpservice/neutron/nsf/configurator/api/root_controller.py

```diff
@@ -0,0 +1,16 @@
+from pecan import expose
+from v1 import controllers
+
+
+class RootController(object):
```

Comment:
default pecan installation does not give this file. But pecan create [project_name] provides root.py file with sample get method. This file provides path to our controller.